.babelrc

```diff
@@ -1,16 +1,9 @@
 {
-  "presets": [
-    "es2015",
-    "stage-3"
-  ],
-  "plugins": [
-    "transform-runtime"
-  ],
+  "presets": ["@babel/es2015", "@babel/stage-3"],
```

Comment:
Could you also replace `preset-es2015` with `preset-env` please?
I've been meaning to do it for a while and that's a good opportunity to do so? :)

Reply:
of course, good idea